### ISSUE TITLE
Merge docs into this repository

### ISSUE THREAD

**Issue by Zauberfisch** [swipestripe/silverstripe-swipestripe#31]
As far as I can tell, it is convention for SilverStripe modules to have a docs folder in the repository.
Until now, SwipeStripe had the docs folder in a separated repository (I guess to make the auto publish on http://swipestripe.com/dev/docs/swipestripe/en easier)

I would like to suggest merging the docs repository into this repository. This will yield several benefits:
- developers that use SwipeStripe get the docs and code in one package
- it is easier to maintain docs if they are part of the repo (Its hard enough already to motivate myself to write docs, I can't be bothered to do it in a separated repository :P)
- it aligns with how its done in SilverStripe and other modules (eg: [framework](https://github.com/silverstripe/silverstripe-framework/tree/master/docs), [gridfieldextensions by ajshort](https://github.com/ajshort/silverstripe-gridfieldextensions/tree/master/docs))

the only downside that I can see is that auto deployment of the docs is a bit more work, but shouldn't really be a problem.

**Comment by frankmullenger** [comment]
Yep, fair enough. I agree with your points here. Not sure why I pulled them out to a separate repo. Perhaps adding docsviewer to the require-dev entry in composer requirements would be a good idea too.
